.gitlab-ci.yml

```diff
@@ -44,7 +72,7 @@ deploy_to_sonatype:
     - set -x
 
     - echo "Building release..."
-    - mvn -DperformRelease=true -Daether.checksums.algorithms=SHA-512,SHA-256,SHA-1,MD5 --settings ./settings.xml clean deploy
+    - mvn -Djdk.attach.allowAttachSelf=true -DperformRelease=true -Daether.checksums.algorithms=SHA-512,SHA-256,SHA-1,MD5 --settings ./settings.xml clean deploy
```

Comment:
Will be necessary in the future if we upgrade the JVM that we perform the releases under to 9+, but currently not needed. Just future-proofing here.